src/e2e/java/teammates/e2e/cases/e2e/BaseE2ETestCase.java

```diff
@@ -59,6 +65,11 @@ protected void releaseBrowser() {
         BrowserPool.release(browser);
     }
 
+    @Override
+    protected String doPutDocuments(DataBundle testData) {
```

Comment:
You should uncomment the code in `BaseTestCaseWithBackDoorApiAccess` rather than overriding the method here?